src/test/scala/chiselTests/Counter.scala

```diff
@@ -41,7 +44,11 @@ class CounterSpec extends ChiselPropSpec {
   }
 
   property("Counter can be en/disabled") {
-    forAll(safeUInts) { (seed: Int) => whenever(seed >= 0) { assertTesterPasses{ new EnableTester(seed) } } }
+    assertTesterPasses{ new EnableTester(4) }
+//    forAll(safeUInts) { (seed: Int) => whenever(seed >= 0) {
```

Comment:
Dead code?
Wait, this actually removes the forall behavior?

Reply:
Good catch, debugging code that slipped through